### PR TITLE
OSDOCS-3053: Removing a KubeletConfig remediation

### PR DESCRIPTION
**Pull request by kmccarron-rh**
https://issues.redhat.com/browse/OSDOCS-3053
Applies to 4.6+

Preview available [here](https://deploy-preview-40343--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-remediation.html#compliance-removing-kubeletconfig_compliance-remediation)

### CODE REVIEW

File: modules/compliance-unapplying-kubeletconfig.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+----
++  
+. Remove the remediation check from the KubeletConfig object by toggling the flag to false:
++
+[source,terminal]
+----
+$ $ oc patch complianceremediations/one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available
+----
+
+. The remediation status will change to `NotApplied` and the composite `KubeletConfig` object will re-render to not include the remediation.
```
Comment (Vincent056) [inline modules/compliance-unapplying-kubeletconfig.adoc:81]: Changing the remediation object to un-apply will not affect `KubeletConfig` object, once you find the name of Kubeletconfig, in this case `compliance-operator-kubelet-master`, you need to `oc edit KubeletConfig compliance-operator-kubelet-master`, and then remove <2> from the edit.

Also to note: if the user has scheduled scan that auto-apply remediation. they would need to exclude the rule they want to unapply in tailored profile, so they don't get applied again.

File: modules/compliance-unapplying-kubeletconfig.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        kubeletConfig:
+          evictionHard:
+            imagefs.available: 10% <2>
+  outdated: {}
+  type: Configuration
+status:
+  applicationState: Applied
+----
+<1> The scan name of the remediation.
+<2> The remediation check for the remediation.
```
Comment (Vincent056) [inline modules/compliance-unapplying-kubeletconfig.adoc:58]: for <2> change to: The remediation that was added to KubeletConfig Objects.

File: modules/compliance-unapplying-kubeletconfig.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+----
+NAME                                 AGE
+compliance-operator-kubelet-master   2m34s
+----
++  
+. Remove the remediation check from the KubeletConfig object by toggling the flag to false:
++
+[source,terminal]
+----
+$ $ oc patch complianceremediations/one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available
```
Comment (Vincent056) [inline modules/compliance-unapplying-kubeletconfig.adoc:78]: as we disscussed, changed `$ $ oc patch complianceremediations/one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available` to `$ oc patch complianceremediations/one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available -p '{"spec":{"apply":false}}' --type=merge`

File: modules/compliance-unapplying-kubeletconfig.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                 AGE
+compliance-operator-kubelet-master   2m34s
+----
++  
+. Remove the remediation check from the KubeletConfig object by toggling the flag to false:
```
Comment (Vincent056) [inline modules/compliance-unapplying-kubeletconfig.adoc:74]: From:

```
Remove the remediation check from the KubeletConfig object by toggling the flag to false:```

Change to something like:

To remove the remediation, first, you need to toggle the flag to false for the remediation object.

**Review by Vincent056 — APPROVED** [review]
Thanks for the PR I think they look good!

**Comment by kmccarron-rh** [comment]
@pdhamdhe - Will you PTAL? Thank you!

File: modules/compliance-unapplying-kubeletconfig.adoc [hunk 5]
```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-unapplying-kubeletconfig_{context}"]
+= Unapplying a KubeletConfig remediation
```
Comment (michaelryanpeter) [inline modules/compliance-unapplying-kubeletconfig.adoc:6]: You might want to consider changing _unapplying_ to something like _removing_ in a future cleanup. _Unapply_ isn't in the American Heritage or Merriam-Webster dictionary, and we don't have specific guidance saying it is okay to use in the the SSG, ISG, or repo guidelines. It is used in the repo, but only for the Compliance Operator docs.

Just something to consider, I won't hold up a merge over it.

**Review by michaelryanpeter — COMMENTED** [review]
Just the one comment to consider. Otherwise, LGTM. Let me know when you are ready for a merge.

File: modules/compliance-unapplying-kubeletconfig.adoc (hunk 5 above)
Comment (kmccarron-rh) [reply modules/compliance-unapplying-kubeletconfig.adoc:6]: @michaelryanpeter  That's a very good point. I'll pass this by dev and see if they agree.

**Pull request merged**